frontend/src/features/SidebarSuite/types.ts

```diff
@@ -0,0 +1,158 @@
+import { AllAPIRequestProps, APISchemas } from "@utils/api/types";
+import type { DbViewEnum } from "@utils/constants";
+
+export type AppResultPageView = DbViewEnum | "search";
+
+type APIRequestPropsName = keyof AllAPIRequestProps;
+
+// `AllAPIRequestPropModel` is an `at a glance` ref that catches any model changes pulled from the API
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const AllAPIRequestPropModel: AllAPIRequestProps = {
```

Comment:
hmm, so the purpose of this is testing if our FE types are still compatible with the API types? It's not actually used anywhere?

I would maybe put it in a test.ts file, but looks good :)